conans/paths.py

```diff
@@ -39,6 +39,8 @@
 
 RUN_LOG_NAME = "conan_run.log"
 
+DEFAULT_PROFILE_NAME = "conan_default"
```

Comment:
"conan" is almost a meaningless name. Why not just "default", or "default_profile"? If you peek into the profiles folder, it would read easier?